logicrunner/builtin/contract/member/member.go

```diff
@@ -241,11 +245,11 @@ func (m *Member) transferCall(params map[string]interface{}) (interface{}, error
 		return nil, fmt.Errorf("failed to get wallet implementation of sender: %s", err.Error())
 	}
 
-	return w.Transfer(amount, toMemberReference)
+	return w.Transfer(amount, recipientReference)
 }
 func (m *Member) migrationCall(params map[string]interface{}) (interface{}, error) {
 
-	inAmount, ok := params["inAmount"].(string)
+	inAmount, ok := params["amount"].(string)
 	if !ok {
 		return nil, fmt.Errorf("incorect input: failed to get 'inAmount' param")
```

Comment:
You try to get `amount ` param, but error message says `inAmount `. Name must be the same.